plugins/callback/logstash.py

```diff
@@ -18,40 +18,85 @@
     options:
       server:
         description: Address of the Logstash server
-        env:
-          - name: LOGSTASH_SERVER
+        version_added: 1.0.0
```

Comment:
Please fix the existing `version_added` entries. Your PR will never get merged if you don't do it. Seriously!

Reply:
I removed `version added` entries from old sections, please check one more time

Reply:
Please take a look at the diff yourself: https://github.com/ansible-collections/community.general/pull/641/files

As long as you can see that version_added has been removed or moved around, you need to fix that. I really don't want to invest any more time in this.